anonymizer/inout.py

```diff
@@ -1,11 +1,14 @@
 import os
 from email import policy
 from email.parser import BytesParser
+from glob import glob
+from pathlib import Path
 
 
-def list_of_files(path):
+def list_of_files(path:Path, ending:str):
```

Comment:
```suggestion
def list_of_files(directory_name: str) -> list[Path]:
    mypath = Path(directory_name)
    pattern = [".eml", ".html"]  # we would not change the file type through user input
    email_list = [ mp.resolve() for mp in mypath.glob("**/*") if mp.suffix in pattern]
```
- we would hardcode the file endings as they are not likely to change, and changing them would break the code on different ends
- you can directly search for several globs using list comprehension (should work like this, have not tested it)
- I would pass a string as it makes it easier for the input directive, and the path object is not needed anywhere else

Reply:
Reasoning was that the different formats might have to be processed in different ways.
This way both formats would end up in the same list.

Reply:
Yes, that's true. But I would not worry about that now. The function currently also does not return a list, the return  from `glob` would need to be converted into a list . Just noticed this.